.github/workflows/pythonpublish.yml

```diff
@@ -7,9 +7,8 @@
 name: Upload Python Package
 
 on:
-  push:
-    branches: 
-      - master
+  schedule:
+    - cron: "0 12 * * *"
```

Comment:
This is the UTC time, I personally prefer 59 8 * * *  which starts 23:59 in PST

Reply:
Good suggestion